pkgs/tools/admin/jetporch/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, rustPlatform,
+  fetchFromSourcehut,
+  git, openssl, gnused,
+  pkg-config,
+  writeText,
+  ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jetporch";
+  version = "0.0.1";
+
+  src = fetchFromSourcehut {
+    owner = "jetporch";
+    repo = "jetporch";
+    rev = "v${version}";
+    hash = "sha256-Gp/HnCM/+1wroC8gLRdm2BY/qadboPQyirYYvL7fED8=";
+  };
```

Comment:
The source is at https://git.sr.ht/~mpdehaan/jetporch now.

Reply:
Fixed in a3b2dcbaa5c0028f086b72dc1d651f28a3c28841

Reply:
The download still does not work. In fact I haven't found any versioned releases in that repo... I think it would be best to use the "unstable-YYYY-MM-DD" version scheme. See https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

Reply:
@FliegendeWurst can you check the amended commit now? Let me know if that works. 

(Thank you for the thoughtful review, btw)